src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { actions as currentTodoAction } from '../../features/currentTodo';
 import { Loader } from '../Loader';
+import { useAppSelector } from '../../app/hooks';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [user, setUser] = useState<User | null>(null);
+
+  const dispatch = useDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    if (currentTodo) {
+      getUser(currentTodo.userId)
+        .then(setUser)
+        .finally(() => setIsLoading(true));
```

Comment:
Consider writing true like the initial value and false into the finally block

Because right now when you get a user you have state isLoading true and I think that you have endless loading

Reply:
no, i don't. There is a test for this case and my loader turning off after promise is resolved